app/frontend/src/views/RequestAccount.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <v-container>
+    <BaseSecure>Request Account</BaseSecure>
+    <RequestForm />
```

Comment:
BaseSecure needs to wrap everything - the way it is right now you can get content leakage. This may be related to the comment from @loneil 
```
<BaseSecure>
  <div>Request Account</div>
  <RequestForm />
</BaseSecure>
```

Reply:
that was the reason. thanks!